e2e/__tests__/multiProjectRunner.test.ts

```diff
@@ -170,36 +170,50 @@ test('"No tests found" message for projects', () => {
   );
 });
 
-test.each([{projectPath: 'packages/somepackage'}, {projectPath: 'packages/*'}])(
```

Comment:
can you add a separate test instead of modifying the old one? Or is the old one wrong somehow?

Reply:
Yup, the environment described by the old test does not sufficiently capture the intended behavior. The old test have 2 configs but only contains only one project (which one wouldn't really know if it runs globally as one project or as multi-project runner as one project).

The new test uses 2 configs + 2 projects.

Reply:
But the old test didn't assert _wrong_ behavior, right? It just asserted a smaller thing? If so, can you keep the old test and add the one you've tweaked as a new one?

Reply:
ok, I've added it back